_benchmark/installing-benchmark.md

```diff
@@ -0,0 +1,154 @@
+| geonames | 11,396,503 | 252.9 MB | 3.3 GB |
+| geopoint | 60,844,404 | 482.1 MB | 2.3 GB |
+| geopointshape | 60,844,404 | 470.8 MB | 2.6 GB |
+| geoshape | 60,523,283 | 13.4 GB | 45.4 GB |
+| http_logs | 247,249,096 | 1.2 GB | 31.1 GB |
+| nested | 11,203,029 | 663.3 MB | 3.4 GB |
+| noaa | 33,659,481 | 949.4 MB | 9.0 GB |
+| nyc_taxis | 165,346,692 | 4.5 GB | 74.3 GB |
+| percolator | 2,000,000 | 121.1 kB | 104.9 MB |
+| pmc | 574,199 | 5.5 GB | 21.7 GB |
+| so | 36,062,278 | 8.9 GB | 33.1 GB |
+
+Your OpenSearch Benchmark host should use solid-state drives (SSDs) for storage because they perform significantly faster at read and write operations than traditional spinning-disk hard drives. Spinning-disk hard drives can introduce performance bottlenecks, which can make benchmark results unreliable and inconsistent.
+{: .tip}
+
+## Installing on Linux and macOS
+
+If you want to run OpenSearch Benchmark in a Docker container, see [Installing with Docker](#installing-with-docker). The OpenSearch Benchmark Docker image includes all of the required software, so there are no additional steps for you to take.
+{: .important}
+
+To install OpenSearch Benchmark directly on a UNIX host such as Linux or macOS, make sure you have **Python 3.8 or newer** installed. 
+
+If you need help installing Python, refer to the official [Python Setup and Usage](https://docs.python.org/3/using/index.html) documentation. 
+
+### Checking software dependencies
+
+Before you being installing OpenSearch Benchmark, check the following software dependencies.
+
+Use [pyenv](https://github.com/pyenv/pyenv) to manage multiple versions of Python on your host. This is especially useful if your "system" version of Python is older than version 3.8.
+{: .tip}
+
+- Check that Python 3.8 or newer is installed:
+
+  ```bash
+  python3 --version
+  ```
+  {% include copy.html %}
+
+- Check that `pip` is installed and functions:
+
+  ```bash
+  pip --version
+  ```
+  {% include copy.html %}
+
+- _Optional_ Check that your installed version of `git` is **Git 1.9 or newer**. `git` is not required for OpenSearch Benchmark installation, but it is required to fetch benchmark workload resources from a repository when you want to perform tests. See the official Git [Documentation](https://git-scm.com/doc) for help installing Git. 
+
+  ```bash
+  git --version
+  ```
+  {% include copy.html %}
+
+### Completing the installation
+
+After the prerequisites are installed, you can install OpenSearch Benchmark using the following command:
+
+```bash
+pip install opensearch-benchmark
+```
+{% include copy.html %}
+
+After the installation completes you can use the following command to display help information:
+
+```bash
+opensearch-benchmark -h
+```
+{% include copy.html %}
+
+OpenSearch Benchmark will exit with an error if no arguments are supplied. Use the `-h` option to print the help text in the command line if you want to test the installation.
```

Comment:
Not sure what you mean here. And does it belong above the command?

Reply:
Removing this line per Ian's comment.